### PR TITLE
SUL23-409 - Update filter on plate to study to design.

### PR DESCRIPTION
**Pull request by mdyoung3**
# READY FOR REVIEW

# Summary
- Design changes to filter: https://stanfordits.atlassian.net/browse/SUL23-409

# Review By (Date)
- Jan 30

# Urgency
- How critical is this PR?
mid

### CODE REVIEW

File: src/components/views/sul-study-place/study-place-filtering.tsx [hunk 1]
```diff
@@ -117,14 +117,18 @@ const StudyPlacesFiltering = ({items}: {items: StudyPlace[]}) => {
             onChange={(event, value: SelectValue<string, boolean>) => setSelectedFeatured(value as string[])}
           />
 
+          <div className="flex flex-col pl-12 mr-80 mt-20">
+            <button type="submit" className="button">
```
Comment (pookmish) [inline src/components/views/sul-study-place/study-place-filtering.tsx:121]: on 1050 wide screen, the submit button acts weird
image

**Review by pookmish — CHANGES_REQUESTED** [review]
a certain breakpoint looks wrong

File: src/components/views/sul-study-place/study-place-filtering.tsx (hunk 1 above)
Comment (mdyoung3) [reply src/components/views/sul-study-place/study-place-filtering.tsx:121]: Fixed!

File: src/components/patterns/elements/select-list.tsx [hunk 2]
```diff
@@ -124,13 +124,13 @@ const SelectList = ({options, label, multiple, ariaLabelledby, ...props}: Props)
...
-        className="w-full border border-black-40 rounded text-left p-5"
+        className="w-full border shadow-md rounded-lg text-left py-7 px-5"
         aria-labelledby={labeledBy}
       >
-        <div className="flex justify-between flex-wrap">
+        <div className="flex flex-wrap">
           {label &&
-            <div className={"relative " + (optionChosen ? "text-m0 top-[-15px] w-full" : "text-m1")}>
-              <div id={labelId} className="bg-white w-fit px-5">
+            <div className={"relative p-10" + (optionChosen ? "top-[-15px] w-full" : "")}>
```
Comment (pookmish) [inline src/components/patterns/elements/select-list.tsx:132]: Missing the space

```suggestion
            <div className={"relative p-10 " + (optionChosen ? "top-[-15px] w-full" : "")}>
```

**Comment by pookmish** [comment]
Looks like this has a negative impact on the home page select lists: 
image

**Comment by mdyoung3** [comment]
@pookmish I need to incorporate backstop before submitting PRs...

**Comment by imonroe** [comment]
Can we close this PR/branch?  It has been superseded by other work, no sense in letting it hang out, right?

**Comment by mdyoung3** [comment]
@imonroe Closed.

**Pull request closed without merging**